iframe/src/mocks/responses/steps/TestGateway/possibleFomStepFields.ts

```diff
@@ -0,0 +1,224 @@
+  },
+  {
+    type: "integer",
+    name: "phoneNumber",
+    humanName: "Phone number",
+    hint:
+      "Landlines and VoIP are not accepted. Please use a mobile phone number",
+  },
+];
+
+const personalInfoFields = [
+  {
+    type: "string",
+    name: "firstName",
+    humanName: "First name",
+  },
+  {
+    type: "string",
+    name: "lastName",
+    humanName: "Last name",
+  },
+  {
+    type: "string",
+    name: "street",
+    humanName: "Street",
+  },
+  {
+    type: "string",
+    name: "town",
+    humanName: "City",
+  },
+  {
+    type: "string",
+    name: "postCode",
+    humanName: "Postal Code",
+  },
+  {
+    type: "date",
+    name: "dateOfBirth",
+    humanName: "Date of Birth",
+    data: [
+      {
+        type: "integer",
+        humanName: "Day",
+        name: "day",
+      },
+      {
+        type: "integer",
+        humanName: "Month",
+        name: "month",
+      },
+      {
+        type: "integer",
+        humanName: "Year",
+        name: "year",
+      },
+    ],
+  },
+  {
+    type: "string",
+    name: "email",
+    humanName: "Email",
+    hint: "Hint for email field.",
+  },
+  {
+    type: "string",
+    name: "identityDocumentLastName",
+    humanName: "Last name as on the identity document",
+  },
+];
+
+const allFormFields = [
+  {
+    type: "string",
+    name: "cryptocurrencyAddress",
+    humanName: "Cryptocurrency wallet address",
+    hint: "This is a test hint for cypto address",
+  },
+  {
+    type: "string",
+    name: "verifyCreditCard",
+    humanName: "Credit Card verification code",
+    hint:
+      "For the time being, most likely I'm not visible because I'm replaced by a text in the frontend!",
+  },
+  {
+    type: "string",
+    name: "verifyPhoneCode",
+    humanName: "Phone verification code",
+    hint: "A verification code was sent to your phone number. Fill it in here.",
+  },
+  {
+    type: "string",
+    name: "verifyEmailCode",
+    humanName: "Email verification code",
+    hint:
+      "This is a test hint for email code which here is required on purpose",
+    isRequired: true,
+  },
+  {
+    type: "select",
+    name: "name for select",
+    humanName: "Select's label",
+    hint: "Select's hint (is required on purpose)",
+    options: [
+      { humanName: "Test option #1", value: "#1" },
+      { humanName: "Test option #2", value: "#2" },
+      { humanName: "Test option #3", value: "#3" },
+    ]
+  },
+  {
+    name: "country",
+    humanName: "Select country",
+    type: "select",
+    placeholder: "A placeholder",
+    options: [
+      { humanName: "Algeria", value: "DZ" },
+      { humanName: "Argentina", value: "AR" },
+      { humanName: "Australia", value: "AU" },
+      { humanName: "United Kingdom", value: "GB" },
+      { humanName: "United States of America", value: "US" },
+      { humanName: "Vietnam", value: "VN" },
+    ]
+  },
+  {
+    type: "string",
+    name: "state",
+    humanName: "Select state",
+    required: false,
```

Comment:
Is it correct that this is not being displayed in the form and that it is included in the payload as `state: "undefined"`.

Reply:
yeah. Forgot to mention that in the description.. you need to add country query parameter when to check this item

Reply:
Great, verified 👍